packages/flutter_tools/test/general.shard/build_system/source_test.dart

```diff
@@ -160,57 +160,52 @@ void main() {
   }));
 
   test('can resolve a missing depfile', () => testbed.run(() {
-    const Source depfile = Source.depfile('foo.d');
+    visitor.visitDepfile('foo.d');
 
-    depfile.accept(visitor);
     expect(visitor.sources, isEmpty);
     expect(visitor.containsNewDepfile, true);
```

Comment:
```suggestion
    expect(visitor.containsNewDepfile, isTrue);
```

Reply:
I generally prefer using the literal over the matcher, though we don't have a rule on way or the  other